the CSS2DObject visible is false but still show

callmegaga opened the pull request:
fixed when the CSS2DObject visible is false,But it still is shown.

Commit:
the CSS2DObject visible is false but still show

fixed when the CSS2DObject visible is false,But it still is shown.

WestLangley commented on examples/js/renderers/CSS2DRenderer.js:
```diff
@@ -82,7 +82,12 @@ THREE.CSS2DRenderer = function () {
+      	element.style.display = 'none'
+      }else {
+      	element.style.display = ''
+      }
```
How about
```js
element.style.display = ( ! object.visible || vector.z < - 1 || vector.z > 1 ) ? 'none' : '';
```

wwwYing replied:
YES,That's better.

Commit:
the CSS2DObject visible is false but still show 

fixed when the CSS2DObject visible is false,But it still is shown.That is better code style

mrdoob commented:
I propose this instead 😇

```js
element.style.display = ( object.visible && vector.z > - 1 && vector.z < 1 ) ? '' : 'none';
```

WestLangley commented:
Then you probably should use `<= 1` and `>= - 1`.

mrdoob commented:
Will merge and amend.

mrdoob commented:
Thanks!
